src/components/ChatList.tsx

```diff
@@ -1,10 +1,28 @@
 import styled from 'styled-components';
 import {useCallback, useState} from 'react';
-import { IChatTypes } from '../Recoil/chat';
+import {chatState, IChatTypes } from '../Recoil/chat';
 import {useRecoilState, useRecoilValue} from 'recoil';
+import ChatItem from './ChatItem';
 
-function ChatList(){
-    return(<div>안녕하세요</div>);
-}
+
+
+const ChatList = (): JSX.Element => {
+    const [chats, setChats] = useRecoilState<IChatTypes[]>(chatState);
+            return (
+                <div>
+                {chats.map((chat: IChatTypes)=>{
+                const {sender, receiver, text, checked} = chat;
+                return(
+                <ChatItem
+                    sender = {sender}
+                    receiver = {receiver}
+                    checked = {checked}
+                    text = {text}
+                />
+                );
+                })}
```

Comment:
`map`을 사용하실 때 가장 상위 컴포넌트에 `key`값을 부여해주셔야해요!
`id`값 등을 추가하시면 좋을 거 같아요 임시 방편으로 length를 지정해봤습니당

```suggestion
{chats.map((chat: IChatTypes)=>{
                const {sender, receiver, text, checked} = chat;
                return(
                <ChatItem
                    key = {text.length}
                    sender = {sender}
                    receiver = {receiver}
                    checked = {checked}
                    text = {text}
                />
                );
                })}
```